hr_employee_calendar_planning/models/resource_calendar.py

```diff
@@ -23,6 +23,21 @@ def _check_active(self):
                     % (item.name, total_items)
                 )
 
+    @api.constrains("company_id")
+    def _check_company_id(self):
+        for item in self.filtered("company_id"):
```

Comment:
Isn't this done automatically through `check_company=True` attribute on the field?

Reply:
Do you mean that it is necessary to add `check_company=True` to the `company_id` field of the `resource.calendar` model?

Reply:
`_check_company_auto=True` is not enough to prevent `company_id` change in resource calendars.